modules/claims_api/app/controllers/claims_api/v1/forms/disability_compensation_controller.rb

```diff
@@ -77,6 +80,7 @@ def upload_form_526
             pending_claim.set_file_data!(documents.first, params[:doc_type])
             pending_claim.save!
 
+            ClaimsApi::Logger.log('526', claim_id: params[:id], detail: 'Uploaded')
```

Comment:
We haven't actually uploaded anything to an upstream VA service at this point.  So the the detail of `Uploaded` is misleading.  All we've done at this point is store a PDF in Amazon S3.  We need a better detail value here.  Something like, `Uploaded PDF to S3`.